### ISSUE TITLE
Rule disabling in code

### ISSUE THREAD

**Issue by jwvirtru** [virtru/lint-trap#24]
http://eslint.org/docs/user-guide/configuring.html

Can we agree that when temporarily disabling ESLint rules in code, we should explicitly name the rule(s) that we are disabling? Being explicit obviously avoids ambiguity when reading code but also prevents unintended violations from being passed over by the linter.

I don't see a way to enforce this through ESLint, so if we adopt this, we'll have to rely on devs and reviewers (or use commit hooks).

Proposed:
```js
// Not OK
/*eslint-disable */
...
/*eslint-enable */

// OK
/*eslint-disable no-console, camelcase */
...
/*eslint-enable no-console, camelcase */
```
@virtru/employees RFC

**Comment by JasonGhent** [comment]
I like this idea a lot.

**Comment by Akkuma** [comment]
http://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments

So the proposed is supported in eslint

**Comment by jwvirtru** [comment]
@Akkuma, yep I've been disabling specific rules in my code. What I'd like for us to consider is disallowing disabling rules via comments without specifying which rules are being disabled (i.e., no more `/*eslint-disable*/` because it doesn't specify what is being disabled).

**Comment by Akkuma** [comment]
I'm going to see is there is a plugin that does this for us otherwise we'll need to create another custom rule
